emails/templates/emails/first_time_user.html

```diff
@@ -231,7 +231,7 @@
             <tbody>
               <tr>
                 <td align="center"> 
-                  <img align="center" border="0" src="{{ SITE_ORIGIN }}/static/images/email-images/first-time-user/hero-image-en.png" alt="" title="" /> 
+                  <img align="center" border="0" src="{{ SITE_ORIGIN }}/static/images/email-images/first-time-user/hero-image-{{request.LANGUAGE_CODE}}.png" alt="" title="" style="width:425px;max-width:425px;"/> 
```

Comment:
This fails for at least `zh-cn`, probably for other supported languages. Attempting to load https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net/static/images/email-images/first-time-user/hero-image-zh-cn.png results in a blank spot in the email:

![PR 3454 - no zh-cn](https://github.com/mozilla/fx-private-relay/assets/286017/f638d743-c1b5-45a5-b43d-bc87b4bff1e9)

I believe `hero-image-cn.png` should be renamed `hero-image-zh-cn.png`, or even `hero-image-zh.png`.

Reply:
Updated to default to `hero-image-en.png` and only change to an l10n image if one is available for the `request.LANGUAGE_CODE`.